app/services/external-links.js

```diff
@@ -1,18 +1,68 @@
 import Service from '@ember/service';
 import config from 'travis/config/environment';
+import { vcsName } from 'travis/helpers/vcs-name';
+
+const lowerVcsName = (vcsType) => (
+  vcsType ? vcsName(vcsType).toLowerCase() : 'github'
+);
```

Comment:
I'd suggest to just assign a default value to the argument:
```js
const lowerVcsName = (vcsType = 'github') => vcsName(vcsType).toLowerCase();
```

Reply:
Also it'd be nice to have all available `vcsType`s in one place, e.g. in dictionary:
```js
export const VCS_TYPES = {
  GITHUB: 'github',
  BITBUCKET: 'bitbucket',
  ASSEMBLA: 'assembla'
}
```
and then reuse them wherever possible:
```js
import { VCS_TYPES } from 'travis/path/to/dict';

const { GITHUB } = VCS_TYPES;
```
this would allow us to avoid hardcoding and simplify refactorings.

Also it would be easy to map `vscName`s:
```js
const { GITHUB, ASSEMBLA } = VCS_TYPES;

export const VCS_NAMES = {
  [GITHUB]: 'GitHub',
  [ASSEMBLA]: 'Assembla'
}
```
and `vcsEndpoints`:
```js
const { GITHUB, ASSEMBLA } = VCS_TYPES;

export const VCS_ENDPOINTS = {
  [GITHUB]: 'https://github.com',
  [ASSEMBLA]: 'https://bitbucket.org'
}
```
What do you think?

Reply:
Totally agree.